shift_detector/precalculations/SorensenDicePrecalculation.py

```diff
@@ -71,6 +71,9 @@ def process(self, store: Store) -> dict:
         df1a, df1b = random_split(df1, [0.95, 0.05], seed=11)           # Baseline for df1
         df2a, df2b = random_split(df2, [0.95, 0.05], seed=11)           # Baseline for df2
 
+        if df1a.empty or df1b.empty or df2a.empty or df2b.empty:
+            raise ValueError('Dataset to small for split ratio or n to big')
```

Comment:
'or n to big' what is n? it should probably state a real number here.

Reply:
It's the **n**gram - n, since you are setting this n also in during the initialization of the check, it should be descriptive enough. I changed it to `or n={} to big` anyway